src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentalPrice = 40;
+  let total = 0;
```

Comment:
This variable is not necessary, you can return your results from inside the if statement instead.